indigo/templates/indigo/admin/joiningupinitiatives.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'indigo/base-admin.html' %}
+
+{% block head_title %}Joining Up Initiatives{% endblock %}
+
+{% block breadcrumbs %}
+    {% include "indigo/admin/joiningupinitiative/includes/breadcrumbs.html" with active="Joining Up Initiatives" %}
+{% endblock %}
+
+{% block content %}
+    <h2 class="mt-5">Joining Up Initiatives</h2>
+    <p class="text-right"><a href="{% url 'indigo_admin_joining_up_initiative_new' %}" class="btn btn-primary">Add Joining Up Initiative <i class="far fa-plus" aria-hidden="true"></i></a></p>
+
+    {% if datas %}
+        <table class="table border-bottom">
+            <tr>
+                <th>ID</th>
+                <th>Name</th>
+                <th>&nbsp;</th>
+            </tr>
+            {% for item in datas %}
+                <tr>
+                    <td>
+                        {% if item.joiningupinitiative_set.exists %}
+                            <a href="{% url 'indigo_admin_joining_up_initiative_index' public_id=item.public_id %}">{{ item.public_id }}</a>
+                        {% else %}
+                            {{ item.public_id }}
+                        {% endif %}
+                    </td>
+                    <td>
+                        {{ item.cached_data.name.value }}
+                    </td>
+                    <td>
+                        {% if item.joiningupinitiative_set.exists %}
+                            <a href="{% url 'indigo_admin_joining_up_initiative_index' public_id=item.public_id %}" class="btn btn-secondary">View</a>
+                            <a href="{% url 'indigo_admin_joining_up_initiative_edit' public_id=item.public_id %}" class="btn btn-primary">Edit</a>
+                        {% else %}
+                            Removed
```

Comment:
Not sure what you're getting at here. I put this in because during development I deleted some joining up initative items, and they were still accessible in the listing. Maybe the way I deleted them was not correct. Anyway, if I create a new (unmoderated) item via the form, it appears at the bottom and not as 'Removed' so in my case exists must be true although not moderated...see INDIGO-JUI-0025
<img width="1040" alt="Screenshot 2023-03-03 at 10 37 13" src="https://user-images.githubusercontent.com/1315296/222702690-ff3b7a3c-4da1-423a-90f7-564bced9b0e1.png">

Reply:
"I deleted some joining up initative items" Ah this I think is the cause of the issue. There isn't really a delete function in the system - it's not been a requirement. 

In rare cases we are able to remove a bit of data and there is purge CLI tools that use https://github.com/INDIGO-Initiative/database-app/blob/live/indigo/purge.py - as you can see they remove multiple objects, and that would have stopped the "still accessible in the listing" issue you saw - I'm guessing you only removed one of the multiple objects that was needed.

Is it a requirement that these will be regularly deleted? If not, and you were just doing it during dev, can we remove the 2 if checks on exists?  It may mislead as that's not what the exists flag does and if we do implement soft delete, I'm pretty sure it will be in a different way.

Reply:
Cool, there is no need for a delete afaik. I'll tidy this up